databook/background/Mouse-visual-system.md

```diff
@@ -39,11 +39,11 @@ organized in columns, the rodent visual cortex has a "salt and pepper"
 organization, where the orientation selectivity of neighboring neurons are
 randomly organized. {cite:p}`ohki`
 
-Beyond {term}`V1` there are additional higher visual areas (or {term}`HVA`s)
-that receive input from V1 and are connected in a hierarchical manner. Higher
-visual areas each contain a distinct map of {term}`Retinotopy`, i.e. each visual
-area has its own map of visual space. The borders between higher visual areas
-can be identified based on reversals in the retinotopic map at area boundaries.
+Beyond {term}`V1` there are additional higher visual areas ({term}`HVA`s)
+which receive input from V1 and are connected in a hierarchical manner. HVAs
```

Comment:
this should remain `that`

Reply:
Perhaps I'm misunderstanding how the cortex works; are there HVAs that do not receive input from V1? I believe correct syntax is "which" if the HVAs receiving input from V1 is a common attribute to them all, and "that" if only some HVAs receive input from V1 and we are clarifying which HVAs we are talking about.

e.g. "....there are HVAs that receive input..." denotes that there are also HVAs that do not receive input, and we are making a comment informing the readers that ones that do receive input exist,

vs. "...there are HVAs which receive input..." is pointing out a further property that all HVAs possess.

Reply:
if you can drop the cluase and leave the meaning of the sentence intact, use which. Here we can't. The input from V1 and hierarchical connection is what defines the HVAs.